numba/ir_utils.py

```diff
@@ -1841,3 +1845,32 @@ def raise_on_unsupported_feature(func_ir):
                             "is being used in an unsupported manner.") % \
                             (use, expr)
                     raise UnsupportedError(msg, stmt.value.loc)
+
+            # this checks for gdb initilization calls, only one is permitted
+            if isinstance(stmt.value, (ir.Global, ir.FreeVar)):
+                val = stmt.value
+                val = getattr(val, 'value', None)
+                if val is None:
+                    continue
+
+                # check global function
+                found = False
+                if isinstance(val, pytypes.FunctionType):
+                    found = val == numba.gdb or val == numba.gdb_init
```

Comment:
i think this statement would be simpler as `found = val in {numba.gdb, numba.gdb_init}`